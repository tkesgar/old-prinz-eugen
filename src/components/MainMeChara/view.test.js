import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import MeCharaListView from './view'

describe('MeCharaListView', () => {
  const wrapper = mount(MeCharaListView, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
