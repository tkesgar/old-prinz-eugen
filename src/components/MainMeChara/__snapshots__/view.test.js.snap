// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeCharaListView should match snapshot 1`] = `
<div>
  <div
    class="text-center"
  >
    <p>
      Belum ada karakter.
    </p>
  </div>
</div>
`;
